web/src/lib/roadmap/roadmapBuilder.ts

```diff
@@ -188,12 +187,12 @@ interface StepBuilder {
   timeEstimate: string;
   section: SectionType;
   description: string;
-  tasks: TaskBuilder[];
 }
 
 interface TaskBuilder {
   filename: string;
   weight: number;
+  step_number: number;
```

Comment:
i know you're just following what we did for Step, but I have no idea why we did snake case instead of camel case for this property. Ideally we'd refactor both to be camel case. Maybe that could be a separate commit though, to separate out the complexity